src/web_resources/HangarXPLOR.DrawUI.js

```diff
@@ -17,13 +63,16 @@ HangarXPLOR.DrawUI = function()
   
   $('.js-pager').remove();
   
-  $controls1.append(HangarXPLOR.Dropdown([
+  // Filter
+  $controls1.append(HangarXPLOR.Dropdown('Filter:', [
     { Value: 'All', Text: 'All Types', Class: 'first', Selected: HangarXPLOR._type == 'All' },
     { Value: 'HasShip', Text: 'All Ships', Selected: HangarXPLOR._type == 'HasShip' },
+    { Value: 'HasShipOrIsUpgradeOrIsAddOn', Text: 'All Ships & Upgrades & Add Ons', Selected: HangarXPLOR._type == 'HasShipOrIsUpgradeOrIsAddOn' },
```

Comment:
I had these at one stage, but the list got very long...

I'm thinking about making it configurable similar to the ribbon bars in office.